docs/certificates.md

```diff
@@ -0,0 +1,54 @@
+# Certificates
```

Comment:
It's great to have this list written down, thanks.  An improvement suggestion:  It would be nice to specify where we only need the certificate and where we need both the certificate and the private key.  For example, national-connector needs the opt-out certificate and FSEU needs the certificate and the private key.